data/reusables/dependabot/supported-package-managers.md

```diff
@@ -116,9 +117,11 @@ For {% data variables.product.prodname_dependabot_security_updates %}, Gradle su
 
 {% ifversion dependabot-PEP621-support %}
 
-#### pip and pip-compile
+#### pip-compile and uv
 
-In addition to supporting updates to `requirements.txt` files, {% data variables.product.prodname_dependabot %} supports updates to `pyproject.toml` files if they follow the PEP 621 standard.
+{% data variables.product.prodname_dependabot %} can update `requirements.txt` compiled files using both `pip-compile` and `uv pip` (depending on which one was used to generate the file), as well as performing updates to `pyproject.toml` files if they follow the PEP 621 standard.
+
+`uv.lock` files are not supported yet.
```

Comment:
```suggestion
`uv.lock` files are not supported.
```

cc @jonjanego in case you want to provide a timeline, otherwise I'd recommend leaving this out.

Reply:
agree, i don't want to imply a roadmap that we cannot promise

Reply:
lets please change this to remove "yet"